packages/parallax/cypress/support/commands.js

```diff
@@ -0,0 +1 @@
+import '@testing-library/cypress/add-commands'
```

Comment:
so could this go to the top level? `root/cypress`

Reply:
Pretty sure that should work. Everything is orchestrated by `start-server-and-test` anyways so as long as the commands fed to that are right it should be fine. 

Or do you mean move the file like `/cypress/commands.js`? (This is also totally possible)

Reply:
see comment above, it'll probably clear it up a bit